wallofslaughter/Player.lua

```diff
@@ -0,0 +1,18 @@
+
+
+function event_enter_zone(e)
+  	if(e.self:HasItem(12080)) then -- Player has Enchated Signet of Disciples on their character and Zones in.
+    		e.self:Message(15, "The Signet begins to glow");
+	end
+end
+
+
+function event_loot(e)
+	if(e.self:Class() == "Warrior") then
+		local qglobals = eq.get_qglobals(e.self);
```

Comment:
Should Player.lua be player.lua?  I think every player file I've seen has been lower case.